docs/logs/extending-the-sdk/extending-the-sdk.csproj

```diff
@@ -1,6 +1,6 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <TargetFrameworks>net461;net462;net47;net471;net472;net48;netcoreapp2.1;netcoreapp3.1</TargetFrameworks>
```

Comment:
we probably dont need all of the targets. show 2.1 as its specially treated. Then one more from .net core 3.1 or 5.0, and one from .net framework. (net461 would be a good candidate as it shows the oldest .net framework supported)

Reply:
Fixed this.